config/routes.rb

```diff
@@ -87,6 +87,7 @@
       get '/account_reset/pending/confirm' => 'account_reset/pending#confirm'
       post '/account_reset/pending/cancel' => 'account_reset/pending#cancel'
 
+      get '/auth-method-confirmation' => 'mfa_confirmation#index'
```

Comment:
For consistency with other routes, can we replace `-` with `_` ?

```suggestion
      get '/auth_method_confirmation' => 'mfa_confirmation#index'
```